code/game/machinery/data_disk.dm

```diff
@@ -192,6 +192,7 @@
 		/datum/design/board/odysseus_peri,
 		/datum/design/board/gygax_main,
 		/datum/design/board/gygax_peri,
+		/datum/design/board/gygax_targ,
```

Comment:
I think this was caused by tg's decision to make *security* have this design, but that's not really possible anymore, so I'm fine with this as-is.